packages/lua53-lpeg/build.sh

```diff
@@ -0,0 +1,37 @@
+TERMUX_PKG_HOMEPAGE=https://www.inf.puc-rio.br/~roberto/lpeg
+TERMUX_PKG_DESCRIPTION="Pattern-matching library for Lua 5.3"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
+TERMUX_PKG_VERSION=1.1.0
+TERMUX_PKG_SRCURL=https://www.inf.puc-rio.br/~roberto/lpeg/lpeg-$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=4b155d67d2246c1ffa7ad7bc466c1ea899bbc40fef0257cc9c03cecbaed4352a
+TERMUX_PKG_BUILD_DEPENDS="liblua51, liblua53"
+TERMUX_PKG_DEPENDS="liblua53"
```

Comment:
As you suggested lua53-lpeg should 
```
TERMUX_PKG_REPLACES="lua-lpeg (<= 1.1.0)"
TERMUX_PKG_BREAKS="lua-lpeg (<= 1.1.0)
``` 

I think that would be enough. Docs: https://wiki.debian.org/RenamingPackages

Reply:
That would also imply `lua53-lpeg` and `lua51-lpeg` need either a revision or epoch bump to be considered greater than 1.1.0?

Reply:
I don't think so, lua53-lpeg 1.1.0 can probably break and replace lua-lpeg <= 1.1.0 without issues. On the other hand it doesn't cost anything to bump revision as well while we are at it!